### ISSUE TITLE
Add (optionally) into the report the list of the failed test (class and name) and a counter with failed age

### ISSUE THREAD

**Issue by db410009** [EnricoMi/publish-unit-test-result-action#186]
When reviewing the report for the failed tests, I miss a "table" including the list of failed test and the "age" for the failure, I mean, how many consecutive failures.

**Comment by EnricoMi** [comment]
So in addition to the test failures listed under annotation on the check page:
![grafik](https://user-images.githubusercontent.com/44700269/141981831-1eaa6713-77cb-40cb-8a0c-399cdc5975c8.png)

You mean to list them in the pull request comment like skipped tests are:
![grafik](https://user-images.githubusercontent.com/44700269/141982154-9ab2ccf0-8755-45d3-b0a8-d3bce9230d82.png)

I think implementing the age feature would be interesting to achieve, but it gives the wrong impression. There should never be tests failing in master. It feels like supporting an anti-use-case. However, contributions are always welcome.

Adding the failed tests as a list is redundant to the list of annotations, which provides more contextual information than just the test and class names. It is just one click away.

**Comment by db410009** [comment]
> So in addition to the test failures listed under annotation on the check page: ![grafik](https://user-images.githubusercontent.com/44700269/141981831-1eaa6713-77cb-40cb-8a0c-399cdc5975c8.png)
> 
> You mean to list them in the pull request comment like skipped tests are: ![grafik](https://user-images.githubusercontent.com/44700269/141982154-9ab2ccf0-8755-45d3-b0a8-d3bce9230d82.png)
> 
> I think implementing the age feature would be interesting to achieve, but it gives the wrong impression. There should never be tests failing in master. It feels like supporting an anti-use-case. However, contributions are always welcome.
> 
> Adding the failed tests as a list is redundant to the list of annotations, which provides more contextual information than just the test and class names. It is just one click away.

Yeah, I was referring to a table/list showing the test suites. It is specially interesting regarding multi module projects.

And related with the "age" and "anti-use-case", I'm almost 100% agreed with you on the bases.
The issue because I miss this is because working on a large project with many components involved and parallel development, we are detecting some tests failures on integration tests. They are not moving to main, but sometimes is hard to identify which is the commit guilty for the failure.

**Comment by AleksanderBrzozowski** [comment]
This could be useful for repository with e2e tests that are "flaky".
For example, in my organization we have a repo with e2e tests that are run every day at night on sandbox environment. We use microservice architecture and sometimes test can fail due to timeout or other reason.
Thus, we do not want to track "single" failures. 
When test fails consecutively it means that some functionality might be broken. And this is what we want to track. Bamboo has this functionality - "new test failures" and "existing test failures". This is the reason why we still use bamboo instead of github actions :)

I think that this feature would require state persistence in order to know if the test failed yesterday or not :)
